Vinyl/Player.swift

```diff
@@ -13,19 +13,19 @@ struct Player {
     let vinyl: Vinyl
     let trackMatchers: [TrackMatcher]
     
-    private func seekTrackForRequest(request: Request) -> Track? {
-        return vinyl.tracks.first { track in
+    fileprivate func seekTrackForRequest(_ request: Request) -> Track? {
+        return vinyl.tracks.first({ track in
```

Comment:
Scratch that. It's only ambiguous because of this:
https://github.com/Velhotes/Vinyl/pull/67/files/48325ee576b3ddc00ee0ea91f83379e1518aa2d4#diff-1dbc32d9dc35d1ccfc62bc0b07dd328cR31
which I'm getting rid of so, problem solved.